pkg/policy/mapstate.go

```diff
@@ -74,24 +79,100 @@ type MapState interface {
 
 // mapState is a state of a policy map.
 type mapState struct {
-	allows mapStateMap
-	denies mapStateMap
+	allows *mapStateMap
+	denies *mapStateMap
+}
+
+type Identities interface {
+	GetNetsLocked(identity.NumericIdentity) []*net.IPNet
 }
 
 // mapStateMap is a convience type representing the actual structure mapping
```

Comment:
`convenience` type?

Reply:
This is actually a git diff artifact from me moving this around. It has not actually been added by this commit.